devicetypes/technisat/technisat-series-switch.src/technisat-series-switch.groovy

```diff
@@ -0,0 +1,440 @@
+/**
+ *	Copyright 2021 TechniSat
+ *
+ *	Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ *	in compliance with the License. You may obtain a copy of the License at:
+ *
+ *		http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *	Unless required by applicable law or agreed to in writing, software distributed under the License is distributed
+ *	on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License
+ *	for the specific language governing permissions and limitations under the License.
+ *
+ */
+import groovy.json.JsonOutput
+
+metadata  {
+	definition (name: "TechniSat Series switch", namespace: "TechniSat", author: "TechniSat", vid:"generic-switch-power-energy",
+				mnmn: "SmartThings", runLocally: true, minHubCoreVersion: '000.017.0012',
+				executeCommandsLocally: false) {
+		capability "Energy Meter"
+		capability "Switch"
+		capability "Power Meter"
+		capability "Refresh"
+		capability "Configuration"
+		capability "Health Check"
+
+		fingerprint mfr: "0299", prod: "0003", model: "1A91", deviceJoinName: "TechniSat Switch 1"
+	}
+
+	preferences {
+		parameterMap.each {
+			input(title: "Parameter ${it.paramZwaveNum}: ${it.title}",
+				description: it.descr,
+				type: "paragraph",
+				element: "paragraph")
+			if(it.enableSwitch) {
+				input(name: it.enableKey,
+					title: "Enable",
+					type: "bool",
+					required: false)
+			}
+			input(name: it.key,
+				title: it.paramName,
+				type: it.type,
+				options: it.values,
+				range: it.range,
+				required: false)
+		}
+	}
+}
+
+private createChild() {
+	
+	log.debug "createChild componentLabel: ${componentLabel}"
+	try {
+		String dni = "${device.deviceNetworkId}-ep2"
+        def componentLabel = "${device.displayName[0..-2]}2"
+		addChildDevice("TechniSat Series switch child", dni, device.getHub().getId(),
```

Comment:
I don't see this DTH anywhere. You'd need to include it in this PR.

Reply:
This request is created via DevWs and it wasn't possible to add a 2nd DTH for 1 product there. So I created the publish request with only the 'main' DTH and immediately after this a 2nd PR including the child DTH which I already linked here (#66242)